.storybook/preview.js

```diff
@@ -56,7 +56,7 @@ const customViewports = {
 
 export const parameters = {
 	viewport: {
-		defaultViewport: 'mobile',
+		defaultViewport: 'tablet',
```

Comment:
<img src="https://user-images.githubusercontent.com/12860328/108697011-354fcb80-74fa-11eb-9a60-60751923d561.gif" alt="choices" width="200" />

Reply:
Not much in it but I kept having to change it - happy to revert if mobile more useful but don't think there's much in it!